src/seedu/addressbook/commands/UpdateCommand.java

```diff
@@ -0,0 +1,58 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.*;
+import seedu.addressbook.data.person.UniquePersonList.PersonNotFoundException;
+import seedu.addressbook.data.tag.Tag;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class UpdateCommand extends Command{
+    public static final String COMMAND_WORD = "update";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Updates details of the person " +
+            "identified by the index number used in the last person listing.\n"
+            + "Parameters: INDEX NAME [p]p/PHONE [p]e/EMAIL [p]a/ADDRESS  [t/TAG]...\n"
+            + "Example: " + COMMAND_WORD +
+            " 1 John Smith p/11111111 e/john@email.com a/21st Orchard Rd, #12-02 Garden Towers t/friends";
+    public static final String MESSAGE_SUCCESS = "%1$s updated.";
+    public static final String MESSAGE_DUPLICATE_DETAILS = "Details provided are the same.";
```

Comment:
Do you actually check for duplicate details being provided in your class? If not, should this message be shown to the user?